### DISCUSSION TITLE
Do we really need the provider component to be added to the project by the chakra cli?

### DISCUSSION THREAD

**Discussion by techwithtwin** [chakra-ui/chakra-ui#9163] [General] [answered]
I don't understand why we need the Provider component generated by the CLI as it ads bloat since we most likely create our own system and add custom stuff like project colors and recipes e.t.c. do we really need it? 

![image](https://github.com/user-attachments/assets/2685c73f-20f9-4ee3-857a-71d474fbf776)

re,
TechWithTwin

**Comment by segunadebayo** [comment, marked as answer]
No, you don't need it. It's just an easier setup.
